specs/networking/p2p-interface.md

```diff
@@ -557,6 +558,14 @@ The Ethereum Node Record (ENR) for an Ethereum 2.0 client MUST contain the follo
 
 Specifications of these parameters can be found in the [ENR Specification](http://eips.ethereum.org/EIPS/eip-778).
 
+#### Shard bitfield
+
+The ENR MAY contain an entry signifying the shard subnet bitfield with the following form to more easily discover peers participating in particular shard gossip subnets.
```

Comment:
This is confusing. Subnets != shards, as per `ATTESTATION_SUBNET_COUNT`. Are we sticking long term to describe peers by their shard they are focusing on? Or the subnet they are on? Which may have more shards (possible if shard count increases), or only be part of a shard (unlikely). If just subnets, let's don't name it `shards`.

Reply:
resolved